purchases/src/main/kotlin/com/revenuecat/purchases/listenerConversions.kt

```diff
@@ -29,8 +29,9 @@ private typealias PurchaseErrorFunction = (error: PurchasesError, userCancelled:
 internal val ON_ERROR_STUB: ErrorFunction = {}
 private val ON_PURCHASE_ERROR_STUB: PurchaseErrorFunction = { _, _ -> }
 
-internal fun purchaseCompletedListener(
-    onSuccess: PurchaseCompletedFunction,
+@Deprecated("onCompleted Purchase changed with PaymentTransaction")
+internal fun deprecatedPurchaseCompletedListener(
```

Comment:
should we move these to the deprecatedListenerConversions file?

Reply:
yes, good point. I will do it in a followup though, so it's easier to review and see what's getting deprecated, like in this comment https://github.com/RevenueCat/purchases-android/pull/343/files#r763477464